packages/playground/src/components/pull-request/interfaces.ts

```diff
@@ -5,7 +5,7 @@ export enum PRCommentFilterType {
   RESOLVED_COMMENTS = 'resolvedComments',
   UNRESOLVED_COMMENTS = 'unresolvedComments'
 }
-export type EnumPullReqReviewDecision = 'approved' | 'changereq' | 'pending' | 'reviewed'
+export type EnumPullReqReviewDecision = 'approved' | 'changereq' | 'pending' | 'reviewed' | 'outdated'
```

Comment:
Why not use existing type here? https://github.com/harness/react-api-client/blob/pkg-update-oct-10/packages/code-service/src/services/schemas/EnumPullReqReviewDecision.ts#L5

If any other value is needed, just extend or add to it.